mollie_subscription_ept/models/sale_order.py

```diff
@@ -9,12 +8,18 @@ class SaleOrder(models.Model):
 
     def _check_subs_in_cart(self):
         result = self.order_line.product_id.filtered(lambda product: product.is_mollie_subscription)
-        return True if result else False
+        res = False
```

Comment:
This is nit working in a single line as return self.order_line.product_id.filtered(lambda product: product.is_mollie_subscription) ?